lib/ex_libsrt/server.ex

```diff
@@ -150,7 +193,33 @@ defmodule ExLibSRT.Server do
   @spec read_socket_stats(connection_id(), t()) ::
           {:ok, ExLibSRT.SocketStats.t()} | {:error, reason :: String.t()}
   def read_socket_stats(connection_id, agent) do
-    server_ref = Agent.get(agent, & &1)
-    ExLibSRT.Native.read_server_socket_stats(connection_id, server_ref)
+    if Process.alive?(agent) do
+      server_ref = Agent.get(agent, & &1)
+      ExLibSRT.Native.read_server_socket_stats(connection_id, server_ref)
+    else
+      {:error, "Server is not active"}
+    end
+  end
+
+  # Private functions
+
+  @spec validate_password(String.t()) :: :ok | {:error, String.t()}
+  defp validate_password(""), do: :ok
+
+  defp validate_password(password) when is_binary(password) do
+    password_length = String.length(password)
+
+    cond do
+      password_length < 10 ->
+        {:error, "SRT password must be at least 10 characters long"}
+
+      password_length > 79 ->
+        {:error, "SRT password must be at most 79 characters long"}
+
+      true ->
+        :ok
+    end
   end
```

Comment:
[NIT] We could move it to some common place so that not to repeat it for server and client, but I don't think it's worth if we don't have any other common code.